### ISSUE TITLE
Faire les étapes dans le désordre

### ISSUE THREAD

**Issue by PomKokca** [edyp-lab/Prostar2#91]
Dans Workflow → Run,
il est possible de faire les étapes dans le désordre. 
Par exemple il est possible de skip l'étape 1, faire la 2 sans save à la fin, puis seulement après faire et save l'étape 1 puis save l'étape 2. 
Les données obtenues après avoir save à l'étape 2 ne prennent pas en compte ce qui a été fait dans l'étape 1. 
Si on continue avec l'étape 3, les données utilisées sont celles obtenues après l'étape 2 (donc sans prise en compte de l'étape 1).

Pour éviter ça peut être qu'il faudrait bloquer les étapes précédentes lorsqu'on clique sur "Start" dans une nouvelle étape, et non pas seulement quand on clique sur "Save" dans la nouvelle étape

**Comment by samWieczorek** [comment]
A voir ensemble... Ce que tu proposes est censé être le comportement de base

**Comment by samWieczorek** [comment]
Effectivement, lorsqu'un process a été validé, il faut invalider (reset) tous les processus qui suivent.
